docs/WR_Desktop/about.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Where's Religion Mobile application
+custom_edit_url: https://github.com/oss-slu/lrda_website
+---
+<!-- A header image is optional; if used should be no greater than 200x600 -->
+<!--![Header Alt Text](header.png) -->
+
+## Overview
+
+![Alt](100x100.png) Where’s Religion is an open-source mobile and desktop application that supports in-person research, remote data entry, media sharing, and mapping. To do this, the mobile app enables users to collect fieldnotes, images, videos, and audio files - all geotagged and timestamped. The desktop companion app provides a more feature-rich format to refine fieldnotes, make new entries, or, for certain user profiles, review or grade other users’ entries.
```

Comment:
I suggest describing the problem this application solves or the value it brings.